OSDOCS#13130: Update the 4.17.14 z-stream RN (#87303)
openshift/openshift-docs: release_notes/ocp-4-17-release-notes.adoc

For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
====

// 4.17.13
[id="ocp-4-17-13_{context}"]
=== RHSA-2025:0351 - {product-title} {product-version}.13 bug fix, and security update advisory

Issued: 21 January 2025

{product-title} release {product-version}.13 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:0351[RHSA-2025:0351] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:0354[RHBA-2025:0354] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.17.13 --pullspecs
----

[id="ocp-4-17-13-bug-fixes_{context}"]
==== Bug fixes

> * Previously, when you clicked the `Don't show again` link in the Red Hat Ansible Lightspeed modal, it did not display the correct `General User Preference` tab when one of the other `User Preference` tabs was open. With this release, clicking the `Don't show again` link goes to the correct `General User Preference` tab. (link:https://issues.redhat.com/browse/OCPBUGS-48227[*OCPBUGS-48227*])

Reviewer: I might switch all of these backticks to bold markup, as GUI elements.

* Previously, the installation program was not compliant with PCI-DSS/BAFIN regulations. With this release, the cross-tenant replication in {azure-first} is disabled, which reduces the chance of unauthorized data access and ensures strict adherence to data governance policies. (link:https://issues.redhat.com/browse/OCPBUGS-48119[*OCPBUGS-48119*])

> * Previously, if `SMTP smarthost` or `SMTP` from fields under `EmailConfigs` were not specified at the global or receiver level in Alertmanager configuration, it caused Alertmanager to crash because they are required fields. With this release, `prometheus-operator` fails reconciliation if these fields are not specified at the global or the receiver level, which avoids pushing an invalid configuration to Alertmanager. (link:https://issues.redhat.com/browse/OCPBUGS-48069[*OCPBUGS-48069*])

Reviewer: Maybe something like this, as I felt like I had to read quite a ways before I understood the scope:
```suggestion
* Previously in Alertmanager configuration, if `SMTP smarthost` or `SMTP` from fields under `EmailConfigs` were not specified at the global or receiver level, it caused Alertmanager to crash because they are required fields. With this release, `prometheus-operator` fails reconciliation if these fields are not specified at the global or the receiver level, which avoids pushing an invalid configuration to Alertmanager. (link:https://issues.redhat.com/browse/OCPBUGS-48069[*OCPBUGS-48069*])
```

* Previously, when you clicked the *Don't show again* link in the Red Hat Ansible Lightspeed modal, it did not display the correct *General User Preference* tab when one of the other *User Preference* tabs was open. With this release, clicking the *Don't show again* link goes to the correct *General User Preference* tab. (link:https://issues.redhat.com/browse/OCPBUGS-48227[*OCPBUGS-48227*])

> * Previously, when a Google Cloud Platform (GCP) service account was created, the account would not always be immediately available. When the account was not available for updates, the installation program received failures when adding permissions to the account. According to Google, https://cloud.google.com/iam/docs/retry-strategy, a service account may be created, but is not active for up to 60 seconds. With this release, the service account is updated on an exponential backoff to give the account enough time to update correctly. (link:https://issues.redhat.com/browse/OCPBUGS-48359[*OCPBUGS-48359*])

Reviewer 2: Probably want to add friendly reference text to this link: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites

> * Previously, on RHEL9 FIPS STIG compliant machines, The SHA-1 key caused the release signature verification to fail because of the restriction to use weak keys.

Reviewer 2: ```suggestion
* Previously, on RHEL 9 FIPS STIG compliant machines, The SHA-1 key caused the release signature verification to fail because of the restriction to use weak keys.
```

> With this release, the key used by oc-mirror for release signature verification is changed and release images are signed by a new SHA256 trusted-key that is different from the old SHA-1 key. (link:https://issues.redhat.com/browse/OCPBUGS-48363[*OCPBUGS-48363*])

Reviewer 2: plugin? 

```suggestion
With this release, the key used by the oc-mirror plugin for release signature verification is changed and release images are signed by a new SHA256 trusted-key that is different from the old SHA-1 key. (link:https://issues.redhat.com/browse/OCPBUGS-48363[*OCPBUGS-48363*])
```

* Previously, the {olm-first} would sometimes concurrently resolve the same namespace in a cluster. This led to subscriptions reaching a terminal state of `ConstraintsNotSatisfiable`, because two concurrent processes interacted with a subscription and this caused a CSV file to become unassociated. With this release, {olm} can resolve concurrent namespaces for a subscription so no CSV remains unassociated. (link:https://issues.redhat.com/browse/OCPBUGS-45845[*OCPBUGS-45845*])

> * Previously, creating a hosted control planes (HCP) cluster using the HCP cli failed on the release image check to verify that the release image was a multi-arch image. With this release, the HCP cli does not fail to create a hosted cluster because the updated cli code with the multi-arch release image check is available. (link:https://issues.redhat.com/browse/OCPBUGS-44927[*OCPBUGS-44927*])

Reviewer 2: ```suggestion
* Previously, creating a hosted control planes (HCP) cluster using the HCP CLI failed on the release image check to verify that the release image was a multi-arch image. With this release, the HCP CLI does not fail to create a hosted cluster because the updated CLI code with the multi-arch release image check is available. (link:https://issues.redhat.com/browse/OCPBUGS-44927[*OCPBUGS-44927*])
```
